Magiczne-Schody/PCAMaster.cpp

```diff
@@ -0,0 +1,54 @@
+#include "stdafx.h"
+#include "PCAMaster.h"
+
+
+PCAMaster::PCAMaster(int howManyStairs, PCA9685 * arrayOfAvaliablePCAs)
+{
+	
+	int numberOfPinsInPCA = 16;
```

Comment:
Proponuję przenieść to do pliku nagłówkowego, mógłby to być prywatny składnik klasy (np. static const)